release-guidelines.md

```diff
@@ -74,7 +74,7 @@ release.
 
 ---
 
-WSO2 Identity and Access Management team is pleased to announce the release of Identity Server <code><em><RELEASED_VERSION></em></code>!
+## WSO2 Identity and Access Management Team Is Pleased To Announce The Release Of Identity Server <code><em><RELEASED_VERSION></em></code>!
 
```

Comment:
IMO the current format is better. @omindu

Reply:
Yes, let's not change this.

Reply:
Addressed in daf7089ecab339128bcd956cdd066cff9c473b5b